.github/CODEOWNERS

```diff
@@ -591,6 +591,7 @@
 /core/templates/pages/teach-page/ @jameesjohn @vojtechjelinek
 /core/templates/pages/thanks-page/ @jameesjohn @vojtechjelinek
 /core/templates/pages/participation-playbook/ @jameesjohn @vojtechjelinek
+/core/templates/pages/license-page/ @jameesjohn @vojtechjelinek
```

Comment:
Move this to core-documentation section and follow the patterns being used for terms-page (line 140) i.e, the HTML file for license page should be owned by @seanlip.

Reply:
done

Reply:
> Move this to core-documentation section

Move this to/around line 133 (in sorted order). [Also, the change on line 141 is correct, thanks! :)]

Reply:
@ReshuKumari is this comment resolved?

Reply:
yes done